src/services/llm/gemini_quote_extractor.py

```
@@ -0,0 +1,230 @@
+            )
+            continue
+
+        extracted.append(sentences[idx])
+
+    # Join sentences with space
+    return " ".join(extracted)
+
+
+def post_process_gemini_response(
+    response_json: dict[str, Any],
+    original_chunks: list[str],
```

Ruff (0.14.5)

Unused function argument: `original_chunks`

(ARG001)